docs/Module-Review/MPEs/CZ4071 Network Science.md

```diff
@@ -0,0 +1,31 @@
+---
+id: cz4071-network-science
+slug: cz4071-network-science
```

Comment:
Can we use the updated course codes that start with `SC`? The MPE list can be found [here](https://www.ntu.edu.sg/docs/librariesprovider118/ug/dsai/list-of-major-prescribed-electives-(dsai---jan-2022).pdf?sfvrsn=85833b7f_3). I believe this is 17th on the list